### PR TITLE
default data location changed to /var/lib/loki

### LINKED ISSUE #7748: Chenge default data location to `/var/lib/loki`
This config file configures Loki to write to `/tmp/loki`. Not sure if this is the ideal default location, but I think it's good enough for a first iteration.

_Originally posted by @chaudum in https://github.com/grafana/loki/pull/6456#discussion_r909328706_

https://github.com/grafana/loki/blob/b4ed595461d595599a72b90b2ae2aa63b7fe6ebb/cmd/loki/loki-local-config.yaml#L7-L12

`/tmp` is usually mounted as [tmpfs](https://en.wikipedia.org/wiki/Tmpfs). All data will be lost after reboot.
Why not use `/var/lib/loki` like [agent](https://github.com/grafana/agent/blob/7de6bb1533c38762f028a556a9bca269f990eecd/packaging/grafana-agent.yaml#L9), [mimir](https://github.com/grafana/mimir/blob/6a8b840b1a61f8080bb5fabb267617034a580086/packaging/deb/systemd/mimir.service#L20) or [grafana](https://github.com/grafana/grafana/blob/23956557d8c6a119b7de5be5c42024e29634d002/packaging/deb/default/grafana-server#L9)?

### PR DESCRIPTION
**Pull request by KanishkaVerma054**
**What this PR does / why we need it**: For ideal default location for config file

**Which issue(s) this PR fixes**:
Fixes #7748

**Special notes for your reviewer**:

**Checklist**
- [x] Reviewed the `CONTRIBUTING.md` guide
- [x] Documentation added
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

File: cmd/loki/loki-local-config.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ server:
   grpc_listen_port: 9096
 
 common:
-  path_prefix: /tmp/loki
+  path_prefix: /var/lib/loki
```
Comment (dannykopping) [inline cmd/loki/loki-local-config.yaml:8]: If you're changing this value, you'll also need to change the `chunks_directory` and `rules_directory` values.

Have you validated that this works?

**Review by dannykopping — COMMENTED** [review]
Thanks for the submission!

File: cmd/loki/loki-local-config.yaml (hunk 1 above)
Comment (KanishkaVerma054) [reply cmd/loki/loki-local-config.yaml:8]: Thanks for the feedback @dannykopping.
May you please clarify me which `chunks_directory` and `rules_directory ` are you refrencing to ?

Comment (dannykopping) [reply cmd/loki/loki-local-config.yaml:8]: Look a few lines down in the config file

Comment (KanishkaVerma054) [reply cmd/loki/loki-local-config.yaml:8]: so I've to change `/tmp/loki/chunks` to `var/lib/loki/chunks` and same for the rules directory?

**Comment by KanishkaVerma054** [comment]
@dannykopping now is the code okay?

File: cmd/loki/loki-local-config.yaml (hunk 1 above)
Comment (owen-d) [reply cmd/loki/loki-local-config.yaml:8]: Yes @KanishkaVerma054 , exactly :)

**Comment by MasslessParticle** [comment]
This appears to be a subset of https://github.com/grafana/loki/pull/7792. I'm closing this one in favor of the other

**Pull request closed without merging**